static/app/components/events/rrwebReplayer/index.tsx

```diff
@@ -1,288 +1,39 @@
-    overflow: hidden;
-    position: relative;
-    box-shadow: inset 0 -1px 3px rgba(0, 0, 0, 0.08);
-  }
-
-  .replayer-mouse,
-  .replayer-mouse:after {
-    z-index: ${p => p.theme.zIndex.tooltip};
-  }
-
-  .rr-controller {
-    width: 100%;
-    display: block;
-    padding: ${space(2)} 0;
-    background: ${p => p.theme.background};
-    border-radius: 0 0 3px 3px;
-    border: 1px solid ${p => p.theme.border};
-    border-top: none;
-    position: relative;
-    color: ${p => p.theme.textColor};
-  }
-
-  .rr-timeline {
-    width: 100%;
-    display: grid;
-    grid-template-columns: 90px 1fr 90px;
-    align-items: center;
-  }
-
-  .rr-timeline__time {
-    text-align: center;
-    color: ${p => p.theme.textColor};
-  }
-
-  .rr-progress {
-    width: 100%;
-    height: 12px;
-    position: relative;
-    cursor: pointer;
-
-    &:before {
-      content: '';
-      background: ${p => p.theme.innerBorder};
-      border-radius: 3px;
-      display: block;
-      height: 6px;
-      position: absolute;
-      left: 0;
-      right: 0;
-      top: 3px;
-    }
-  }
-
-  .rr-progress.disabled {
-    cursor: not-allowed;
-  }
-
-  .rr-progress__step {
-    height: 6px;
-    position: absolute;
-    left: 0;
-    top: 3px;
-    border-radius: 3px;
-    background: ${p => p.theme.purple200};
-  }
-
-  .rr-progress__handler {
-    width: 20px;
-    height: 20px;
-    border-radius: 50%;
-    position: absolute;
-    top: 6px;
-    transform: translate(-50%, -50%);
-    background: ${p => p.theme.purple300};
-  }
-
-  .rr-controller__btns {
-    display: grid;
-    grid-auto-flow: column;
-    grid-auto-columns: max-content;
-    gap: ${space(0.75)};
-    align-items: center;
-    justify-content: center;
-    font-size: ${p => p.theme.fontSizeSmall};
-  }
-
-  .rr-controller__btns button {
-    color: ${p => p.theme.textColor};
-    width: 28px;
-    height: 28px;
-    display: flex;
-    padding: 0;
-    align-items: center;
-    justify-content: center;
-    background: none;
-    border: none;
-    border-radius: 50%;
-    cursor: pointer;
-    transition: background 200ms ease;
-
-    > svg {
-      fill: ${p => p.theme.textColor};
+import {useEffect, useState} from 'react';
+import * as Sentry from '@sentry/react';
+import type RRWebPlayer from 'rrweb-player';
+
+import BaseRRWebReplayer from './baseRRWebReplayer';
+
+type RRWebEvents = ConstructorParameters<typeof RRWebPlayer>[0]['props']['events'];
+interface Props {
+  urls: string[];
+  className?: string;
+}
+
+function RRWebReplayer({urls}: Props) {
+  const [events, setEvents] = useState<RRWebEvents>();
+
+  const loadEvents = async () => {
+    try {
+      const data = await Promise.all(
+        urls.map(async url => {
+          const resp = await fetch(url);
+          const json = await resp.json();
+
+          return json.events;
+        })
+      );
+
+      // `data` is Array<Array<RRWebEvent>> that we will want to flatten
+      setEvents(data.flat());
```

Comment:
Would a more appropriate comment be something like?

```suggestion
      // Merge the Array<Array<RRWebEvent>> into a contiguous list of RRWebEvent's      
      setEvents(data.flat());
```

Reply:
I added a more extensive explanation above the fetch